website/source/api/system/sealwrap-rewrap.html.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: "api"
+page_title: "/sys/sealwrap/rewrap - HTTP API"
+sidebar_title: "<code>/sys/sealwrap/rewrap</code>"
+sidebar_current: "api-http-system-sealwrap-rewrap"
+description: |-
+  The `/sys/sealwrap/rewrap` endpoint is used to rewrap all seal wrapped entries.
+---
+
+# `/sys/sealwrap/rewrap`
+
+~> **Enterprise Only** – These endpoints require Vault Enterprise.
+
+The `/sys/sealwrap/rewrap` endpoint is used to rewrap all seal wrapped entries.
+
+## Read Rewrap Status
+
+This endpoint retrieves if a seal rewrap process is currently running.
+
+| Method   | Path                   |
+| :------- | :--------------------- |
+| `GET`    | `/sys/sealwrap/rewrap` |
+
+### Sample Request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    http://127.0.0.1:8200/v1/sys/sealwrap/rewrap
+```
+
+### Sample Response
+
+```json
+{
+  "request_id": "a6a51003-2576-be0b-9a43-d3bdfeafc2f7",
+  "lease_id": "",
+  "lease_duration": 0,
+  "renewable": false,
+  "data": {
```

Comment:
We usually only include what's the `data` object in `Sample Response`s.

Reply:
I didn't know that. I will fix it. Thanks!